creating_images/sti_testing.adoc

```diff
@@ -143,45 +143,54 @@ a new STI builder image.
 ----
 {
   "kind": "BuildConfig",
-  "apiVersion": "v1beta1",
-  "metadata":{
+  "apiVersion": "v1",
+  "metadata": {
     "name": "ruby-20-centos7-build"
   },
-  "triggers": [
-    {
-      "type": "github",
-      "github": {
-        "secret": "secret101"
+  "spec": {
+    "triggers": [
+      {
+        "type": "github",
+        "github": {
+          "secret": "secret101"
+        }
       }
-    },
-  ],
-  "parameters": {
-    "source" : {
-      "type" : "Git",
-      "git" : {
+    ],
+    "source": {
+      "type": "Git",
+      "git": {
         "uri": "git://github.com/openshift/sti-ruby.git"
       }
     },
     "strategy": {
       "type": "Custom",
       "customStrategy": {
-        "image": "openshift/sti-image-builder",
-        "exposeDockerSocket": true,
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "sti-image-builder:latest"
```

Comment:
I think we want to advertise our ImageStreams here as much as possible, the example as is is working just fine and getting information that you're missing some IS is perfectly ok when someone tries to use this as an example, imho.